tools/azure-sdk-tools/devtools_testutils/proxy_testcase.py

```diff
@@ -58,36 +62,39 @@ def get_recording_assets(test_id: str) -> str:
 
     return None
 
+
 def start_record_or_playback(test_id: str) -> "Tuple[str, Dict[str, str]]":
     """Sends a request to begin recording or playing back the provided test.
 
     This returns a tuple, (a, b), where a is the recording ID of the test and b is the `variables` dictionary that maps
     test variables to values. If no variable dictionary was stored when the test was recorded, b is an empty dictionary.
     """
     variables = {}  # this stores a dictionary of test variable values that could have been stored with a recording
-    
+
     json_payload = {"x-recording-file": test_id}
     assets_json = get_recording_assets(test_id)
     if assets_json:
-        json_payload["x-recording-assets-file"] = assets_json    
+        json_payload["x-recording-assets-file"] = assets_json
 
     if is_live():
-        result = requests.post(
-            RECORDING_START_URL,
-            json=json_payload,
+        result = http_client.request(
+            method="POST",
+            url=RECORDING_START_URL,
+            fields=json_payload,
```

Comment:
After testing, using `fields` doesn't seem to work. Json encoding and using `body` does work fine though:

i.e.:
```
json_payload = json.dumps(json_payload).encode('utf-8')
result = http_client.request(
    method="POST",
    url=RECORDING_START_URL,
    body=json_payload
)
```

I would replace all usage of `fields` in `http_client.request` to use `body`.